### PR TITLE
First router version

### PR DESCRIPTION
**Pull request by guillaumedbk**

### CODE REVIEW

File: src/Router/Controller/HomeController.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+use App\Router\Request;
+
+class HomeController
+{
+    private string $titi;
+
+    public function __construct(string $titi)
+    {
+        $this->titi = $titi;
+    }
```
Comment (cacahouwete) [inline src/Router/Controller/HomeController.php:14]: Why this construct ? Where do you use $titi ?

File: Exceptions/ControllerNotFoundException.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        parent::__construct('Controller not found');
+        $this->request = $request;
+    }
+
+    public function getRequest(): Request
+    {
+        return $this->request;
+    }
+}
```
Comment (cacahouwete) [inline Exceptions/ControllerNotFoundException.php:20]: All php code should be in src folder no ?

**Review by cacahouwete — CHANGES_REQUESTED** [review]

File: src/Router/Controller/HomeController.php (hunk 1 above)
Comment (guillaumedbk) [reply src/Router/Controller/HomeController.php:14]: You're right, it was unused, just deleted it

File: Exceptions/ControllerNotFoundException.php (hunk 2 above)
Comment (guillaumedbk) [reply Exceptions/ControllerNotFoundException.php:20]: Indeed, it's changed

**Pull request merged**